packages/shared/src/offlineScreen/OfflineScreen.js

```diff
@@ -19,40 +20,51 @@ type Props = {|
 
 export default function OfflineScreen(props: Props) {
   return (
-    <View style={styles.container}>
-      <Text style={styles.title}>
-        <Translation id="shared.offline_screen.offline_title" />
-      </Text>
-      <Text style={styles.text}>
-        <Translation id="shared_offline_screen.offline_text" />
-      </Text>
-      <View style={styles.imageContainer}>
-        <Image source={OfflineImage} />
+    <SafeAreaView style={styles.flexItem}>
+      <View style={styles.container}>
+        <View style={styles.flexItem} />
```

Comment:
Is this one necessary?

Reply:
Yes, it is to create the offset above the text, so it looks ok on both tablet and mobile